Makefile

```diff
@@ -1,9 +1,21 @@
 
 # Image URL to use all building/pushing image targets
 IMG ?= openkruise/kruise-manager:test
+# Platforms to build the image for
+PLATFORMS ?= linux/amd64,linux/arm64,linux/arm
+# Automatically push the resulting image
+# Must be set to true if using a build cluster
+# See BUILDING-IMAGE.md for details
+AUTOPUSH ?= false
 # Produce CRDs that work for API servers that supports v1beta1 CRD and conversion, requires k8s 1.13 or later.
 CRD_OPTIONS ?= "crd"
 
+ifeq (true,$(AUTOPUSH))
+DESTARG=--push
+else
+DESTARG=--load
```

Comment:
It seems `--load` can only work for single-platform image https://github.com/docker/buildx#--load

How about only use `--push` for `docker buildx build`?

Reply:
Done.